src/config.js

```diff
@@ -0,0 +1,7 @@
+/* global process */
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
question is there a reason to alias this?

Reply:
@paulschreiber For me this is clearer to understand in its usage, if we just leave it as v4 then the usage will be `v4()` and that is really confusing from a redability point of view